Tasks/DotNetCoreInstallerV0/dotnetcoreinstaller.ts

```diff
@@ -138,17 +204,48 @@ class DotnetCoreInstaller {
         return cachedDir;
     }
 
+    /**
+     * Search for files from a given path.
+     * @param dir the dir we use to search for files
+     * @param searchPattern the regex search pattern the file name must match. The default matches on all names!
+     * @param recursive search recursive for the files. The default is true!
+     */
+    private getFiles(dir: string, searchPattern?: string, recursive?: boolean): Array<string> {
```

Comment:
There is a utility in task lib which you can use to find matching files: [findMatch](https://github.com/Microsoft/azure-pipelines-task-lib/blob/master/node/docs/azure-pipelines-task-lib.md#taskfindMatch)

Reply:
this is the recommended way

Reply:
I think I have resolve this. If not reopen this conversion.